src/components/member-profile/contribution-type/contributions-type.module.scss

```diff
@@ -42,3 +42,27 @@
     margin: 10px;
   }
 }
+
```

Comment:
why are we being particular with `px` and not using `rem`, for any specific reasons?